core/schemas/__init__.py

```diff
@@ -0,0 +1,120 @@
+import importlib
+import inspect
+import logging
+from pathlib import Path
+
+import aenum
+
+from core.schemas import entity, indicator, observable
+
+logger = logging.getLogger(__name__)
+
+
+def load_entities():
+    logger.info("Registering entities")
+    modules = dict()
+    for entity_file in Path(__file__).parent.glob("entities/**/*.py"):
+        if entity_file.stem == "__init__":
+            continue
+        logger.info(f"Registering entity type {entity_file.stem}")
+        if entity_file.parent.stem == "entities":
+            module_name = f"core.schemas.entities.{entity_file.stem}"
+        elif entity_file.parent.stem == "private":
+            module_name = f"core.schemas.entities.private.{entity_file.stem}"
+        enum_value = entity_file.stem.replace("_", "-")
```

Comment:
This should be documented somewhere, maybe in the private README.md docs?

Reply:
what details would you like to document? the `entity_file.stem.replace("_", "-")` part?

Reply:
fixed in d809ec22da543f18f088472547ea26d502ee8ea7